packages/api/src/router/user.ts

```diff
@@ -126,20 +131,33 @@ export const userRouter = createTRPCRouter({
         })
         .where(eq(users.id, input.userId));
     }),
-  getAll: publicProcedure.query(async ({ ctx }) => {
-    return await ctx.db.query.users.findMany({
-      columns: {
-        id: true,
-        name: true,
-        email: true,
-        emailVerified: true,
-        image: true,
-        provider: true,
-      },
-    });
-  }),
-  selectable: publicProcedure.query(async ({ ctx }) => {
-    return await ctx.db.query.users.findMany({
+  getAll: publicProcedure
+    .input(z.void())
```

Comment:
Is input needed here?

Reply:
No, but the openapi library requires ``input`` and ``output``